tests/services/phase.service.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { of } from 'rxjs';
+import { SessionData } from '../../src/app/core/models/session.model';
+import { Phase, PhaseService } from '../../src/app/core/services/phase.service';
+
+const moderationPhaseSettingsFile: {} = {
+  'openPhases': [Phase.phaseModeration],
+  [Phase.phaseBugReporting]: 'bugreporting',
+  [Phase.phaseTeamResponse]: 'pe-results',
+  [Phase.phaseTesterResponse]: 'testerresponse',
+  [Phase.phaseModeration]: 'pe-evaluation'
+};
+
+const invalidPhasesSettingsFile: {} = {
+  ...moderationPhaseSettingsFile,
+  'openPhases': ['dummyPhase']
+};
+
+const multipleOpenPhasesSettingsFile: {} = {
+  ...moderationPhaseSettingsFile,
+  'openPhases': [Phase.phaseBugReporting, Phase.phaseTeamResponse]
+};
+
+let phaseService: any;
+let githubService: any;
+
+describe('PhaseService', () => {
+  beforeEach(() => {
+    githubService = jasmine.createSpyObj('GithubService',
+      ['fetchSettingsFile', 'storePhaseDetails']);
+    phaseService = new PhaseService(null, githubService, null, null, null);
+  });
+
+  describe('.storeSessionData()', () => {
+    it('should return an Observable of true if an openPhase is defined', () => {
+      githubService.fetchSettingsFile.and.returnValue(of(moderationPhaseSettingsFile));
+      githubService.storePhaseDetails.and.callFake(() => {});
+      phaseService.storeSessionData().subscribe((result: boolean) => {
+        expect(result).toBeTruthy();
+      });
+    });
+
+    it('should return an Observable of true if multiple openPhases are defined', () => {
+      githubService.fetchSettingsFile.and.returnValue(of(multipleOpenPhasesSettingsFile));
+      githubService.storePhaseDetails.and.callFake(() => {});
+      phaseService.storeSessionData().subscribe((result: boolean) => {
+        expect(result).toBeTruthy();
```

Comment:
```suggestion
        expect(result).toBeTrue();
```
Any reason for using `toBeTruthy` / `toBeFalsy` over `toBeTrue` / `toBeFalse` ?

Reply:
I made a typo! psps